openstack/workflow/v2/executions/requests.go

```diff
@@ -0,0 +1,44 @@
+package executions
+
+import "github.com/gophercloud/gophercloud"
+
+// CreateOptsBuilder allows extension to add additional parameters to the Create request.
+type CreateOptsBuilder interface {
+	ToExecutionCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts specifies parameters used to create an execution.
+type CreateOpts struct {
```

Comment:
It looks like the following fields are missing:

* https://github.com/openstack/mistral/blob/master/mistral/api/controllers/v2/execution.py#L229-L242
* https://github.com/openstack/mistral/blob/master/mistral/api/controllers/v2/execution.py#L244
* https://github.com/openstack/mistral/blob/master/mistral/api/controllers/v2/execution.py#L262

Reply:
Fixed